sdk/search/search-documents/src/serviceModels.ts

```diff
@@ -381,7 +381,7 @@ export interface PatternAnalyzer {
   lowerCaseTerms?: boolean;
   /**
    * A regular expression pattern to match token separators. Default is an expression that matches
-   * one or more whitespace characters. Default value: '\W+'.
+   * one or more whitespace characters. Default value: '\\W+'.
```

Comment:
This feels a little confusing, since someone reading through the source comments might think it's literally `\\W+`. Is there some way we can get out of escaping this backslash?

Reply:
Changed the ```'\W+'``` to ``` `\W+` ```. This resolved the issue.